health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/NewCampaignCreate/AppModule.js

```
@@ -29,23 +138,59 @@
       </div>
       <div className="modules-container">
         {modulesData?.map((module, index) => (
-          <Card className={"module-card"}>
-            <HeaderComponent className={"detail-header"}>{t(module.code)}</HeaderComponent>
+          <Card className={`module-card ${selectedModuleCodes.includes(module?.name) ? "selected-card" : ""}`}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)